### PR TITLE
fix(style): TDX-1421 a11y style fixes for kong theme

### PR DESCRIPTION
**Pull request by codelemur**
### Summary

*NB*: I will be squashing this to get rid of the "wip" commits.

This is a bulk fix for the following items being tracked in Kong's internal JIRA:

#### kong-portal-templates [style]
- [X] Adjust colors or other styles of HTTP method badges
- [X] Adjust color of section description text
- [X] Adjust color of navigation links
- [X] Adjust color of sidebar endpoint descriptions
- [X] Adjust color of version label
- [X] Adjust color of sign up button
- [X] Adjust color of description links
- [X] Adjust color of footer brand name
- [X] Adjust color of select

### Implementations

1. Mostly involved using updated Kong colors with higher contrast ratios. Lots of changes here and there, but some overrides of the swagger-ui theme.

### Issues Resolved

1. fix TDX-1422
2. fix TDX-1423
3. fix TDX-1424
4. fix TDX-1425
5. fix TDX-1426
6. fix TDX-1427
7. fix TDX-1428
8. fix TDX-1429
9. fix TDX-1430

### CODE REVIEW

**Comment by codelemur** [comment]
Method Badges:
image

Section Description, description links, version label, 
image

Navigation Links, Signup Button
image

Footer
image

File: workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css [hunk 1]
```diff
@@ -337,6 +338,11 @@
 
 .swagger-ui .info {
   word-break: break-word !important;
+
```
Comment (nateslo) [inline workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css:341]:
```suggestion
```

nitpick

File: workspaces/default/themes/base/assets/styles/site.css [hunk 2]
```diff
@@ -652,12 +666,18 @@ a.tag-link.is-active {
...
+  /* steel-200 */
+  background: #dae3f2;
   display: inline-block;
 }
 
+.homepage-clients .client a {
+  font-weight: 500;
+  /* steel-600 */
+  color: #395380;
+  text-decoration: none;;
```
Comment (Mierenga) [inline workspaces/default/themes/base/assets/styles/site.css:678]: nit: double semicolon

File: workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css [hunk 3]
```diff
@@ -134,6 +134,7 @@
   overflow: hidden;
 }
 .sidebar-list ul li .submenu-items a {
+  color: #1456cb;
```
Comment (Mierenga) [inline workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css:137]: I know this color value was hard-coded before, but I think we should use a CSS variable for this color, like:
```css
:root {
  --link-color: #1456cb;
}
```

File: workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css [hunk 4]
```diff
@@ -1057,6 +1071,34 @@
 .swagger-ui .opblock-body .right-side-wrapper pre span.token.italic{font-style:italic !important}
 .swagger-ui .opblock-body .right-side-wrapper pre span.token.entity{cursor:help !important} */
 
+
+.swagger-ui .opblock-summary-get.opblock-summary .opblock-summary-method {
+  background-color: #1456cb;
+}
+
+.swagger-ui .opblock.opblock-delete .opblock-summary-method {
```
Comment (Mierenga) [inline workspaces/default/themes/base/assets/styles/swagger-ui-kong-theme.css:1079]: Can we use a selector consistent with the others here, like `.swagger-ui .opblock-summary-delete.opblock-summary .opblock-summary-method`?

**Comment by nateslo** [comment]
image

image


Note sure if we want to handle it in this PR, but the it looks like the sidebar method colors are still using the old color

**Comment by codelemur** [comment]
@nateslo You guessed it, I've got an ancillary PR coming for the `swagger-ui-kong-theme`[PR here](https://github.com/Kong/swagger-ui-kong-theme/pull/49)

**Comment by nateslo** [comment]
> @nateslo You guessed it, I've got an ancillary PR coming for the `swagger-ui-kong-theme`[PR here](https://github.com/Kong/swagger-ui-kong-theme/pull/49)

Ok cool. Unless @Mierenga has any other comments, this looks good to me.

**Review by Mierenga — APPROVED** [review]
Nice job, looks like we got all the style issues that Axe reports! 🪓🌴

**Pull request merged**